ouroboros-consensus/docs/Testing.md

```diff
@@ -307,6 +307,19 @@ far behind the wallclock, we might not have sufficient knowledge to translate
 the wallclock to a `SlotNo`, although we _can_ always translate the `SlotNo`
 at the tip of the chain to a `UTCTime`.
 
+
+## Golden tests
+
+Golden tests are specific to each era:
+
+- [`Test.Consensus.Byron.Golden`](ouroboros-consensus-byron-test/test/Test/Consensus/Byron/Golden.hs)
+- [`Test.Consensus.Cardano.Golden`](ouroboros-consensus-cardano-test/test/Test/Consensus/Cardano/Golden.hs)
+- [`Test.Consensus.Shelley.Golden`](ouroboros-consensus-shelley-test/test/Test/Consensus/Shelley/Golden.hs)
+
+When adding a new golden test, running the test suite locally will generate the
+golden files. These files should be checked in as CI will fail if there are
+missing golden files.
```

Comment:
We might want to mention that this is done via the `--no-create` flag (which, as we discovered some time ago, is opt-in, surprisingly), which is set here:

https://github.com/input-output-hk/ouroboros-network/blob/a3a9c62256e8fb7f73cd1db7d1b652cae07396ec/nix/ouroboros-network.nix#L56-L62

In particular, this has to be adapted when we introduce golden tests in more tests suites.

Reply:
Exactly, 2e93b229b298e9ed4a794e69b4ef6b33e100cb8c looks good :+1: